reconcile/utils/saasherder.py

```diff
@@ -1634,8 +1646,14 @@ def _get_subscribe_saas_file_path_map(saas_files, auto_only=False):
                     subscribe = target_promotion.get("subscribe")
                     if not subscribe:
                         continue
+                    target_path = target.get("path")
```

Comment:
for future "us" leave a comment stating that if target has a path it is a dedicated file
it seems obvious now but might be hard to read in the future